Jenkinsfile_nightly

```diff
@@ -30,7 +30,9 @@ def integrationTestSecrets = [
   'fpl-aat': [
     secret('integration-test-notify-service-key', 'INTEGRATION_TEST_NOTIFY_SERVICE_KEY'),
     secret('docmosis-api-key', 'INTEGRATION_TEST_DOCMOSIS_TORNADO_KEY'),
-    secret('e2e-test-password', 'E2E_TEST_PASSWORD')
+    secret('e2e-test-password', 'E2E_TEST_PASSWORD'),
+    secret('system-update-user-username', 'SYSTEM_UPDATE_USER_USERNAME'),
```

Comment:
@prabhamuthu15 do these need to be added to README.md for .env info for people?

Reply:
Updated in Readme